From 8e265ebcd60688ae68cb4c75f7d82c5e984951f6 Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Thu, 28 May 2015 21:52:53 +0800
Subject: [PATCH 0629/1221] MLK-10992 ARM: dts: imx6: update the model string
 of imx6qp

For i.MX6QP, the value of '/sys/devices/soc0/machine' should be
"Freescale i.MX6 Quad Plus SABRE Automotive Board". Update the
model string for i.MX6QP.

Signed-off-by: Bai Ping <b51503@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabreauto.dts |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qp-sabreauto.dts b/arch/arm/boot/dts/imx6qp-sabreauto.dts
index 85260e6..bd6262f 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto.dts
@@ -9,6 +9,10 @@
 #include "imx6q-sabreauto.dts"
 #include "imx6qp.dtsi"
 
+/ {
+	model = "Freescale i.MX6 Quad Plus SABRE Automotive Board";
+};
+
 &mxcfb1 {
 	prefetch;
 };
-- 
1.7.5.4

